src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -264,6 +280,16 @@ public function outputLabelsPdf(): void {
 				__( 'Label printing failed, you can find more information in the Packeta log.', 'packeta' ) :
 				__( 'You selected orders that were not submitted yet', 'packeta' );
 			$this->messageManager->flash_message( $message, MessageManager::TYPE_ERROR );
+
+			$redirectTo = $this->httpRequest->getQuery( PacketActionsCommonLogic::PARAM_REDIRECT_TO );
+			if ( PacketActionsCommonLogic::REDIRECT_TO_ORDER_DETAIL === $redirectTo && null !== $idParam ) {
+				try {
+					$this->packetActionsCommonLogic->redirectTo( $redirectTo, $this->orderRepository->getById( (int) $idParam ) );
```

Comment:
Načtení objednávky s ignorováním výjimky už používáme hodněkrát. Co na to zavést metodu v repozitáři?

Reply:
Tato metoda by se používala jen tam, kde není pro účel prováděné akce důležité, proč se nepodařilo instanci objednávky získat. Tedy tam, kde je prázdná catch větev.

Určitě přidat další, jsou i místa, kde se výjimka používá. Pojmenování by mělo vzniknout v rámci realizace, napadá mě například getByIdWithSuppressedInvalidCarrierException, což je trochu dlouhé, ale výstižné.

Reply:
Může být, to řeší ten dlouhý název.